app-foundation/4-projects/shared/infra_pipeline.tf

```diff
@@ -42,9 +42,10 @@ module "app_infra_cloudbuild_project" {
 }
 
 module "infra_pipelines" {
-  source                = "github.com/terraform-google-modules/terraform-example-foundation/4-projects/modules/infra_pipelines"
-  cloudbuild_project_id = module.app_infra_cloudbuild_project.project_id
-  billing_account       = var.billing_account
-  default_region        = var.primary_location
-  app_infra_repos       = ["boa-infra"]
+  source                   = "github.com/terraform-google-modules/terraform-example-foundation/4-projects/modules/infra_pipelines"
+  cloudbuild_project_id    = module.app_infra_cloudbuild_project.project_id
+  billing_account          = var.billing_account
+  default_region           = var.primary_location
+  app_infra_repos          = ["boa-infra"]
+  terraform_apply_branches = ["development", "non-production", "production", "shared"]
```

Comment:
i think it should be `non\\-production`, can you do a quick test?
https://github.com/terraform-google-modules/terraform-example-foundation/blob/508cf48be951ff1fcaf374cb600276861ed9e732/0-bootstrap/main.tf#L149

Reply:
changed for now @bharathkkb, will do a full test early morning for 5-shared CB, along with this leave it up to you to merge or keep PR open

Reply:
we can merge since we know it works (since it does for bootstrap, can you open an issue to track future removal)